README.md

```diff
@@ -208,46 +258,37 @@ python src/lbaf/Utils/lbsVTDataExtractor.py
 * `check_schema`: bool - when True, validates schema (more time-consuming)
 
-## LBAF in Literature
+</details>
 
-### [Optimizing Distributed Load Balancing for Workloads with Time-Varying Imbalance](https://ieeexplore.ieee.org/document/9556089)
+<details>
+<summary><b>VT Data Maker</b></summary>
 
-"This paper explores dynamic load balancing algorithms used by asynchronous many-task (AMT), or ‘taskbased’, programming models to optimize task placement for scientific applications with dynamic workload imbalances."
-
-[//]: # (## Getting Started with Docker)
-
-[//]: # (### Example use:)
-
-[//]: # ()
-[//]: # (Replace `<in_dir>` with path to existing directory which will be mapped with `/lbaf/in` in container)
-
-[//]: # ()
-[//]: # (Replace `<out_dir>` with path to existing directory which will be mapped with `/lbaf/out` in container)
-
-[//]: # (```shell)
-
-[//]: # (docker run -it -v "<out_dir>:/lbaf/out" -v "<in_dir>:/lbaf/in" nganalytics/lbaf "python src/Applications/NodeGossiper.py -l /lbaf/data/vt_example_lb_stats/stats -x 4 -y 2 -z 1 -s 0 -f 4 -k 4 -i 4 -c 1 -e" "/bin/bash")
+<br />
 
-[//]: # (```)
+The VT Data Maker generates a dataset of JSON files that may be used throughout the DARMA-tasking organization. The generated files are compatible with `LBAF`, `vt-tv`, and `vt`.
 
-[//]: # (### Example use explained:)
+If the `lbaf` package is installed, run with:
 
-[//]: # (- container starts with interactive mode &#40;stdout visible&#41;)
+```sh
+lbaf-vt-data-files-maker <args>
+```
 
-[//]: # (- two volumes are mounted&#40;data exchange between host and container possible&#41;:)
+Otherwise, run:
 
-[//]: # (  - directory `<in_dir>` on the host and `/lbaf/in` is mount inside container)
+```shell
+python src/lbaf/Utils/lbsJSONDataFilesMaker.py <args>
+```
 
-[//]: # (  - directory `<out_dir>` on the host and `/lbaf/out` is mount inside container)
+The program can be run interactively with the `--interactive` argument.
 
-[//]: # (- docker image `nganalytics/lbaf`)
+Otherwise, it accepts a pre-written specification file (`--spec-file`) and the file stem for the resulting data files (`--data-stem`).
 
-[//]: # (- commands executed inside container:)
+Further documentation, including usage and examples, can be found within the script itself.
 
-[//]: # (  - sample LBAF usage:)
+</details>
 
-[//]: # (    ```"python src/Applications/NodeGossiper.py -l /lbaf/data/vt_example_lb_stats/stats -x 4 -y 2 -z 1 -s 0 -f 4 -k 4 -i 4 -c 1 -e"```)
+## LBAF in Literature
 
-[//]: # (  - command to stay inside container, after above command is completed:)
+### [Optimizing Distributed Load Balancing for Workloads with Time-Varying Imbalance](https://ieeexplore.ieee.org/document/9556089)
 
-[//]: # (    ```"/bin/bash"```)
+"This paper explores dynamic load balancing algorithms used by asynchronous many-task (AMT), or ‘taskbased’, programming models to optimize task placement for scientific applications with dynamic workload imbalances."
```

Comment:
`task-based' seems better

Reply:
Fixed (I will note that this is taken directly from the paper)